gef-ui/server/controllers/unissued-facilities/index.test.js

```diff
@@ -205,24 +204,41 @@ describe('postChangeUnissuedFacility()', () => {
     '/gef/application-details/123/unissued-facilities');
   });
 
-  it('should not update facility if incorrect start and end dates', async () => {
+  it('should not update facility if no name or dates', async () => {
     mockRequest.body.facilityType = 'CASH';
-    mockRequest.body.facilityName = 'Foundry4';
-    mockRequest.query.saveAndReturn = 'true';
-
-    mockRequest.body['issue-date-day'] = format(tomorrow, 'd');
-    mockRequest.body['issue-date-month'] = format(tomorrow, 'M');
-    mockRequest.body['issue-date-year'] = format(tomorrow, 'yyyy');
+    mockRequest.body.facilityId = 'xyz';
 
-    mockRequest.body['cover-start-date-day'] = format(threeMonthsAndOneDayFromNow, 'd');
-    mockRequest.body['cover-start-date-month'] = format(threeMonthsAndOneDayFromNow, 'M');
-    mockRequest.body['cover-start-date-year'] = format(threeMonthsAndOneDayFromNow, 'yyyy');
+    await postChangeUnissuedFacility(mockRequest, mockResponse);
 
-    mockRequest.body['cover-end-date-day'] = format(oneYearFromNow, 'd');
-    mockRequest.body['cover-end-date-month'] = format(oneYearFromNow, 'M');
-    mockRequest.body['cover-end-date-year'] = format(oneYearFromNow, 'yyyy');
+    // console.log(mockResponse.render);
```

Comment:
remove but can be done after this PR is merged as it's blocking others :D

Reply:
removed.  Had linting error anyways so removed as part of fix